tests/blackbox/stratisd_cert.py

```diff
@@ -136,39 +137,107 @@ def _unittest_command(self, result, expected_return_code):
 
         self.assertEqual(return_code, expected_return_code, msg=msg)
 
+    def _test_permissions(self, dbus_method, permissions, *args):
+        """
+        Test running dbus_method with and without root permissions.
+        :param dbus_method: StratisDbus wrapper method
+        :param bool permissions: True if dbus_method needs root permissions to succeed.
+                                False if dbus_method should succeed without root permissions.
+        """
+        _ROOT = 0
+        _NON_ROOT = 1
+        _permissions_flag = False
```

Comment:
What is the best way to deal with the snake case error. Should I move the constants elsewhere, rename them, or ignore pylint?